tests/AuthorizedAccessTokenControllerTest.php

```diff
@@ -18,6 +19,11 @@ class AuthorizedAccessTokenControllerTest extends TestCase
      */
     protected $tokenRepository;
 
+    /**
+     * @var \Mockery\Mock|Illuminate\Events\Dispatcher
```

Comment:
```suggestion
     * @var \Mockery\Mock|\Illuminate\Events\Dispatcher
```

Reply:
Ready, I changed to use the Dispatcher contratc.